.github/workflows/cargo-tests.reusable.yaml

```
@@ -60,11 +60,27 @@ env:
+    #   bs-8vcpu-j16 — 8-vCPU, -j16 (oversubscribe; sccache hits are I/O-bound)
+    #   bs-16vcpu    — 16-vCPU, default -j16
+    name: "cargo test (linux, ${{ matrix.id }})"
+    runs-on: ${{ matrix.runs-on }}
+    timeout-minutes: 40
+    env:
+      CARGO_BUILD_JOBS: ${{ matrix.jobs }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { id: bs-8vcpu-j8,  runs-on: blacksmith-8vcpu-ubuntu-2404,  jobs: "8" }
```

actionlint (1.7.12)

[error] label "blacksmith-8vcpu-ubuntu-2404" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows", "blacksmith-16vcpu-ubuntu-2404", "blacksmith-8vcpu-ubuntu-2204-arm", "blacksmith-4vcpu-ubuntu-2404", "blacksmith-4vcpu-ubuntu-2204", "blacksmith-4vcpu-ubuntu-2204-arm", "blacksmith-4vcpu-windows-2025". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)